src/clients/ntp-client_proposal.rb

```diff
@@ -8,6 +8,8 @@ module Yast
   class NtpClientProposalClient < Client
     include Yast::Logger
 
+    REQUIRED_PACKAGE ||= "chrony".freeze
```

Comment:
this is wrong as it means it is needed only when client is redefined. It should be solved more by this way - https://github.com/yast/yast-installation/blob/master/src/lib/installation/clients/copy_files_finish.rb#L349